tests/parser/features/external_contracts/test_external_contract_calls.py

```diff
@@ -53,13 +53,13 @@ def array() -> bytes <= 3: pass
 def bar(arg1: address) -> num:
     return Foo(arg1).foo()
```

Comment:
For `Foo(arg1).foo()` type calls. Should we consider enforcing modifiable and static keywords perhaps?
Because it might seem inconsintent to the 'declared address on storage approach'.
`Foo(arg1, 'static').foo()` or `Foo(arg1, type='static').foo()` would be my syntax suggestions.

Reply:
Neither of those is great. What about `modifiable(Foo(arg1).Foo())`? This is a sort of secondary syntax (preferring to explicitly set), so I'm okay with it being uglier